WBooks/New Group/Library/LibraryViewController.swift

```diff
@@ -7,3 +7,90 @@
 //
 
 import Foundation
+import UIKit
+import WolmoCore
+
+class LibraryViewController: UIViewController {
+    private let _libraryView: LibraryView = LibraryView.loadFromNib()!
+    
+    var booksArray: Array = [["tittle": "A Little Bird Told Me", "author":"Timothy Cross", "img":"img_book1"], ["tittle": "When the Doves Disappeared", "author":"Sofi Oksanen", "img":"img_book2"],["tittle": "The Best Book in the World", "author":"Peter Sjernstrom", "img":"img_book3"], ["tittle": "Be Creative", "author":"Tony Alcazar", "img":"img_book4"], ["tittle": "Redesign the Web", "author":"Liliana Castilla", "img":"img_book5"]]
+    
+    override func loadView() {
+        view = _libraryView
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setUpCells()
+        setUpNavigation()
+    }
+    
+}
+
+extension LibraryViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return booksArray.count
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 100
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = _libraryView.booksTable.dequeue(cell: LibraryTableViewCell.self, for: indexPath)!
+        
+        let dict = booksArray[indexPath.row]
+        
+        cell.titleLabel.text = dict["tittle"]
+        cell.authorLabel.text = dict["author"]
+        cell.frontBookImage.image = UIImage(named: dict["img"]!)
+        cell.bodyBook.layer.cornerRadius = 10
+        
+        return cell
+    }
+    
+}
+
+fileprivate extension LibraryViewController {
+    func setUpCells() {
+        _libraryView.booksTable.register(cell: LibraryTableViewCell.self)
+        _libraryView.backgroundColor = UIColor.backgroundLightBlue()
+        _libraryView.booksTable.delegate = self
+        _libraryView.booksTable.dataSource = self
+    }
+    
+    func setUpNavigation() {
+        let backgroundImg = UIImage(named: "bc_nav bar")
+        
+        navigationController?.navigationBar.setBackgroundImage(backgroundImg, for: .default)
+        
+        let searchImg = UIImage(named: "ic_search")
```

Comment:
make an extension for UIImage so you can call them without worrying about the asset name. For example :
```
public static let search() -> UIImage {
       return UIImage(named: "ic_search")!
}
```

and then you use it like


```
UIBarButtonItem(image: .search(), style: .done... etc)
```